test/fhirtypes/StructureDefinition.test.ts

```diff
@@ -120,11 +152,24 @@ describe('StructureDefinition', () => {
   });
 
   describe('#toJSON', () => {
-    // Skipping because the differential doesn't come back right.
-    // Need to re-evaluate how we do differentials.
-    it.skip('should round trip back to the original JSON', () => {
+    it('should round trip back to the original JSON (excepting differentials)', () => {
+      const oldJSON = cloneDeep(jsonObservation);
       const newJSON = observation.toJSON();
-      expect(newJSON).toEqual(jsonObservation);
+      // Because of the way we do differentials, those won't come back the same,
+      // so remove differentials before comparing.
```

Comment:
Yes, that's basically it. We only calculate differentials for what changed since we populated the SD. Although, TBH, I didn't think that much about it. This test was entirely _skipped_ before because the differentials failed the test -- so I unskipped it, verified that it still fails, and did what you see here so that we'd at least have _part_ of a test.